WebCryptoAPI/generateKey/successes.js

```diff
@@ -70,6 +70,18 @@ function run_test(algorithmNames, slowTest) {
         }, testTag + ": generateKey" + parameterString(algorithm, extractable, usages));
     }
 
+    // Only test a subset of tests with, e.g., ?1-10 in the URL
+    var subTestStart = 0;
+    var subTestEnd = Infinity;
+    var match;
+    if (location.search) {
+        match = /^\?(\d+)-(\d+|last)$/.exec(location.search);
+        subTestStart = match[1];
```

Comment:
What happens if the match operation fails here?

Reply:
I made it run all tests.